### DISCUSSION TITLE
Does Parcels support releasing particles from the same set of locations but at different rates?

### DISCUSSION THREAD

**Discussion by Caiyuan6** [Parcels-code/Parcels#1482] [Q&A]
Hello, everyone. I wonder if it is possible to release particles at varying rates from the same set of locations in Parcels.   If this feature is supported, could you guide me on how to implement it?

**Comment by erikvansebille** [comment]
Dear @Cyuan-hub, you can simply give lists of longitudes, latitudes and start dates when you create a particle set, using `ParticleSet.from_list()`, see the [API here](https://docs.oceanparcels.org/en/latest/reference/particles.html#parcels.particleset.ParticleSet.from_list)

So for each particle you want to release, just define its lon, lat, time (and depth, if you don't want the default zero depth)

**Reply by Caiyuan6** [reply]
Hi, @erikvansebille, thank you very much.
